cuda_core/tests/conftest.py

```diff
@@ -7,10 +7,23 @@
 # is strictly prohibited.
 
 from cuda.core.experimental._device import Device

```

Comment:
I think `Device` is public API, no need to access it through a private module:
```suggestion
from cuda.core.experimental import Device
```